### ISSUE TITLE
Are LaTeX equations supported ?

### ISSUE THREAD

**Issue by vincent-picaud** [JuliaPluto/static-export-template#9]
Hi there !
First of all thank you for this work !
I started to use it with the aim of publishing some algorithm demos and short blog posts (https://vincent-picaud.github.io/Some_Pluto_notebooks/AutoDiff_Slides/pluto_ad_introduction.html by example).
However my LaTeX equations are not properly printed (we only see LaTeX code).
Is it the “normal” behavior ? Is there a fix ?
Best,
Vincent

**Comment by Pangoraw** [comment]
Hi, thanks for reporting! This is an issue that only affects exports made with Pluto version 0.18.2 (see https://github.com/fonsp/Pluto.jl/issues/1987). Before a new version is published you can pin the Pluto version to the previous one in the export workflow by adding this line to the `.github/workflows/ExportPluto.yaml` file:

```diff
                  Pkg.add([
+                    Pkg.PackageSpec(name="Pluto", version = "0.18.1"),
                     Pkg.PackageSpec(name="PlutoSliderServer", version="0.3.2-0.3"),
                  ])
```

At this location in the file:

https://github.com/JuliaPluto/static-export-template/blob/6d7db32abeedeffdd5da32eaf171f47793fc1a2e/.github/workflows/ExportPluto.yaml#L37-L39

**Comment by vincent-picaud** [comment]
Hi Pangoraw,

I just modified my `.github/workflows/ExportPluto.yaml` file as you suggested and .... 
that works !!! 

Thank you @Pangoraw for your quick answer :)

**Comment by fonsp** [comment]
We released Pluto 0.18.4 with a fix, so you can go back to the old setup!
